mujoco_warp/_src/solver_test.py

```diff
@@ -325,6 +325,33 @@ def cost(m, d, qacc):
         "efc_force2",
       )
 
+  def test_frictionloss(self):
```

Comment:
this is decent, but also consider adding back frictionloss to `constraints.xml` as it is in MJX - that might test more edge cases.

Reply:
dof frictionloss is restored to `constraints.xml`

Reply:
`test_frictionloss` now tests the first two keyframes from `constraints.xml`.

note: the tests do not pass with the third keyframe with or without frictionloss. a todo has been added.